torch/__init__.py

```diff
@@ -966,3 +965,9 @@ def _register_device_module(device_type, module):
     from . import library
     if not TYPE_CHECKING:
         from . import _meta_registrations
+
+final_all = []
```

Comment:
Why is this necessary? didn't we update this file to not have any of these already?